twitchio/client.py

```diff
@@ -480,6 +486,46 @@ async def fetch_tags(self, ids: List[str] = None):
         data = await self._http.get_stream_tags(ids)
         return [models.Tag(x) for x in data]
 
+    async def fetch_streams(
+        self,
+        user_ids: List[str] = None,
```

Comment:
user_ids should be List[int], not List[str]

Reply:
Have made the change. Thanks!